client/app/views/app_layout.coffee

```diff
@@ -87,10 +87,15 @@ module.exports = class AppLayout extends Mn.LayoutView
     # Regions contents rendering
     # ##########################################################################
     onRender: ->
-        @showChildView 'toolbar', new ToolbarView()
+        toolbar = new ToolbarView()
+        @listenToOnce toolbar, 'show', ->
```

Comment:
It's weird. You don't use the same syntax for this call to `listenToOnce` than for other calls. Shouldn't it be:

```coffee
@listenToOnce toolbar, show: ->
```